dist/linux/appimage/build.sh

```diff
@@ -33,6 +32,7 @@ ${JAVA_HOME}/bin/jlink \
 
 # create app dir
 envsubst '${SEMVER_STR} ${REVISION_NUM}' < ../launcher-gtk2.properties > launcher-gtk2.properties
+TRAYICONSDIR="@{appdir}/usr/share/icons/hicolor/symbolic/apps"
```

Comment:
Can be inlined here as well

Reply:
That's true. Should be changed here as well. Done with 9e25d0fcc04f29b2745ad8b3f46e04bb4af3df38.